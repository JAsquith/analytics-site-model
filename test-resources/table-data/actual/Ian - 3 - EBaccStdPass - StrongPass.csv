Name,EBacc Strong Pass
Ethnic Code,2
Compare With Set (Assessments) >,0
Difference >,2
Ethnic Code,0
Compare With Set (Assessments) >,0
Difference >,0
Ethnic Code,5
Compare With Set (Assessments) >,0
Difference >,5
Ethnic Code,3
Compare With Set (Assessments) >,
Difference >,
Ethnic Code,1
Compare With Set (Assessments) >,0
Difference >,1
Ethnic Code,2
Compare With Set (Assessments) >,0
Difference >,2
Ethnic Code,3
Compare With Set (Assessments) >,0
Difference >,3
FSM Ever 6,9
Compare With Set (Assessments) >,0
Difference >,9
FSM Ever 6,7
Compare With Set (Assessments) >,0
Difference >,7
Gender,3
Compare With Set (Assessments) >,0
Difference >,3
Gender,13
Compare With Set (Assessments) >,0
Difference >,13
Gifted Talented,9
Compare With Set (Assessments) >,0
Difference >,9
Gifted Talented,7
Compare With Set (Assessments) >,0
Difference >,7
KS2 Banding,7
Compare With Set (Assessments) >,0
Difference >,7
KS2 Banding,3
Compare With Set (Assessments) >,0
Difference >,3
KS2 Banding,6
Compare With Set (Assessments) >,0
Difference >,6
Looked After,8
Compare With Set (Assessments) >,0
Difference >,8
Looked After,8
Compare With Set (Assessments) >,0
Difference >,8
SEN,4
Compare With Set (Assessments) >,0
Difference >,4
SEN,4
Compare With Set (Assessments) >,0
Difference >,4
SEN,8
Compare With Set (Assessments) >,0
Difference >,8
Summary,16
Compare With Set (Assessments) >,0
Difference >,16