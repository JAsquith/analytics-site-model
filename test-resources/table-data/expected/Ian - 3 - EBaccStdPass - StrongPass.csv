Name,Filter Value,EBacc Strong Pass
Ethnic Code,MOTH,2
Compare With Set (Assessments) >,,0
Difference >,,2
Ethnic Code,MWAS,0
Compare With Set (Assessments) >,,0
Difference >,,0
Ethnic Code,MWBC,5
Compare With Set (Assessments) >,,0
Difference >,,5
Ethnic Code,REFU,3
Compare With Set (Assessments) >,,
Difference >,,
Ethnic Code,WBRI,1
Compare With Set (Assessments) >,,0
Difference >,,1
Ethnic Code,WIRI,2
Compare With Set (Assessments) >,,0
Difference >,,2
Ethnic Code,WIRT,3
Compare With Set (Assessments) >,,0
Difference >,,3
FSM Ever 6,N,9
Compare With Set (Assessments) >,,0
Difference >,,9
FSM Ever 6,Y,7
Compare With Set (Assessments) >,,0
Difference >,,7
Gender,Female,3
Compare With Set (Assessments) >,,0
Difference >,,3
Gender,Male,13
Compare With Set (Assessments) >,,0
Difference >,,13
Gifted Talented,N,9
Compare With Set (Assessments) >,,0
Difference >,,9
Gifted Talented,Y,7
Compare With Set (Assessments) >,,0
Difference >,,7
KS2 Banding,Lower,7
Compare With Set (Assessments) >,,0
Difference >,,7
KS2 Banding,Middle,3
Compare With Set (Assessments) >,,0
Difference >,,3
KS2 Banding,Upper/High,6
Compare With Set (Assessments) >,,0
Difference >,,6
Looked After,N,8
Compare With Set (Assessments) >,,0
Difference >,,8
Looked After,Y,8
Compare With Set (Assessments) >,,0
Difference >,,8
SEN,E,4
Compare With Set (Assessments) >,,0
Difference >,,4
SEN,K,4
Compare With Set (Assessments) >,,0
Difference >,,4
SEN,N,8
Compare With Set (Assessments) >,,0
Difference >,,8
Summary,,16
Compare With Set (Assessments) >,,0
Difference >,,16
